scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -313,6 +314,20 @@
 	 */
 	private CipherSuiteSelector cipherSuiteSelector;
 
+	/**
+	 * Preselected cipher suites.
+	 * 
+	 * If no supported cipher suites are provided, consider only this subset of
+	 * {@link CipherSuite} to be selected as supported cipher suites for this
+	 * peer.
```

Comment:
```suggestion
	 * If no supported cipher suites are provided, consider only this subset of
	 * {@link CipherSuite} to be automatically selected as supported cipher suites depending 
	 * on other setting (e.g. if settings allow only PSK, only PSK compatible cipher suite from this list will be selected). 
```

Maybe this deserve a bit more explanation.
I propose something but feel free to adapt or just keep it like this. :slightly_smiling_face: